chat-plugins/modlog.js

```diff
@@ -6,7 +6,10 @@ const ProcessManager = require('./../process-manager');
 const execFileSync = require('child_process').execFileSync;
 
 const MAX_PROCESSES = 1;
-const RESULTS_MAX_LENGTH = 100;
+const DEFAULT_RESULTS_LENGTH = 100;
+const MORE_BUTTON_INCREMENTS = [200, 400, 800, 1600, 3200];
+const LINES_SEPARATOR =	'lines=';
```

Comment:
Also, weird, this is what this looked like on my text editor:
<img src="https://image.prntscr.com/image/7AEs7mbKRoyj6Q3h-TUd2Q.png">